README.md

```diff
@@ -6,7 +6,7 @@ Run [`elm-review`] from Node.js.
 
 ## Installation
 
-```bash
```

Comment:
What's the difference in practice? Asking because I always do `bash`.

Reply:
Purely semantic, as far as I can tell. I wrote this two months ago, so 🤷‍♂️
I don't think it makes a difference to syntax highlighting (on gh at least), etc, but I use zsh, Simon uses fish, etc.
I just always use `sh` for shell stuff unless I need a specific shell.